lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -2009,7 +2003,7 @@ LLVMToSPIRVBase::transValueWithoutDecoration(Value *V, SPIRVBasicBlock *BB,
     assert(BV);
     // Don't store opaque pointer constants in the map--we might reuse the wrong
```

Comment:
Perhaps comment should be updated to:

//  Don't store pointer constants in the map -- they are opaque and thus we might reuse the wrong